README.md

```diff
@@ -1,284 +1,55 @@
 
-```python
-import pydarn
-import matplotlib.pyplot as plt
+Python data visualization library for the Super Dual Auroral Radar Network (SuperDARN).
 
-fitacf_file = "./20181209.C0.sas.fitacf"
-darn_read = pydarn.DarnRead(fitacf_file)
-fitacf_data = darn_read.read_fitacf()
+## Changelog
 
-plt.subplot(4, 1, 1)
-plt.title("Summary style plot")
-pydarn.RTP.plot_range_time(fitacf_data, parameter='elv',
-                           beam_num=7, date_fmt='')
-plt.subplot(4, 1, 2)
-pydarn.RTP.plot_range_time(fitacf_data, parameter='p_l',
-                           beam_num=7, date_fmt='')
+## Version 1.0 - Release!
 
-plt.subplot(4, 1, 3)
-pydarn.RTP.plot_range_time(fitacf_data, parameter='v',
-                           beam_num=7, cmap='jet_r',
-                           date_fmt='')
+pyDarn is released! Included are the following features:
+- Reading and writing DMap format IQDAT, RAWACF, FITACF, GRID/GRD and MAP files
+- Reading and writing HDF5 format files for Borealis radar data, as well as conversion to and from DMap format
+- Range-time parameter style plots for RAWACF and FITACF files
+- Summary plots for RAWACF and FITACF files
+- Time series plots for RAWACF and FITACF files
 
-plt.subplot(4, 1, 4)
-pydarn.RTP.plot_range_time(fitacf_data,
-                           parameter='w_l',
-                           beam_num=7)
+## Documentation
 
-plt.show()
-```
-![pyDARN rang-time plot us](./pydarn_range_time_plots.png)
+pyDARN's documentation can found [here](https://pydarn.readthedocs.io/en/latest)
 
-##### Using matplotlib axes object
+## Getting Started
 
-```python
-import pydarn
-import matplotlib.pyplot as plt
+To install and use pyDARN please read the [installation guide](https://pydarn.readthedocs.io/en/latest/user/install/).
 
-fitacf_file = "./20181209.C0.sas.fitacf"
-darn_read = pydarn.DarnRead(fitacf_file)
-fitacf_data = darn_read.read_fitacf()
+If wish to get access to SuperDARN data please read [SuperDARN data access documentation](https://pydarn.readthedocs.io/en/latest/user/superdarn_data/).
+Please make sure to also read the documentation on [**citing superDARN and pydarn**](https://pydarn.readthedocs.io/en/latest/user/citing/). 
 
-fig, (ax1, ax2) = plt.subplots(2, 1)
-pydarn.RTP.plot_range_time(fitacf_data, parameter='elv',
-                           beam_num=7, ax=ax1, channel=1,
-                           date_fmt='')
-ax1.set_ylabel("ch. 1")
+As a quick tutorial on using pydarn to read a compressed file: 
+``` python
+import bz2
+import pydarn 
 
-pydarn.RTP.plot_range_time(fitacf_data, parameter='elv',
-                           beam_num=7, ax=ax2,
-                           channel=2)
-ax2.set_xlabel("Date (UTC)")
-ax2.set_ylabel("ch. 2")
+# read in compressed file
+fitacf_file = '20180220.C0.rkn.stream.fitacf.bz2'
+with bz2.open(fitacf_file) as fp: 
+      fitacf_stream = fp.read()
```

Comment:
I wouldn't mind opinions from others if this is the common way they open files? From my understanding, most people will uncompress the file before trying to read it. So these instructions might only apply for a minority.

Additionally, tutorials on reading in files can be found in the documentation, so this might be better suited there instead of clogging up the readme.

Reply:
I want to have one example to get people started just in case they are lazy about reading the documentation. We can switch it to a more general case if reading compressed files seems less common. That is why I said there is more in the documentation for tutorials to help lead users to it.